src/retyped/jbuild

```diff
@@ -2,4 +2,4 @@
 
 (library
   ((name retyped)
-    (libraries (sedlex flow_parser))))
+    (libraries (js_of_ocaml sedlex flow_parser))))
```

Comment:
I don't think we can use stuff from 4.04 yet 😐 But implementing `split` was perfect 😄